presto-native-tests/src/test/java/com/facebook/presto/nativetests/AbstractTestQueriesNative.java

```diff
@@ -0,0 +1,1174 @@
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testApplyLambdaRepeated() {}
+
+    /// Queries in this testcase use the apply function, which is used to test lambda expressions, and is currently
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testLambdaCapture() {}
+
+    /// Queries in this testcase use the apply function, which is used to test lambda expressions, and is currently
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testLambdaInAggregationContext() {}
+
+    /// Queries in this testcase use the apply function, which is used to test lambda expressions, and is currently
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testLambdaInSubqueryContext() {}
+
+    /// Queries in this testcase use the apply function, which is used to test lambda expressions, and is currently
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testNonDeterministicInLambda() {}
+
+    /// Queries in this testcase use the apply function, which is used to test lambda expressions, and is currently
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testRowSubscriptInLambda() {}
+
+    /// Queries in this testcase use the apply function, which is used to test lambda expressions, and is currently
+    /// unsupported in Presto C++. See issue: https://github.com/prestodb/presto/issues/20741.
+    @Override
+    @Test(enabled = false)
+    public void testTryWithLambda() {}
+
+    /// This test is not applicable in Presto C++ as there is no bytecode IR as with JVM.
+    @Override
+    @Test(enabled = false)
+    public void testLargeBytecode() {}
+
+    /// The functions returned by SHOW FUNCTIONS and their order differ in Presto and Presto C++. Exact comparison is
+    /// not needed and this test is disabled in Presto C++.
+    @Override
+    @Test(enabled = false)
+    public void testShowFunctions() {}
+
+    /// This test uses char type which is not supported in Presto C++. See issue:
+    /// https://github.com/prestodb/presto/issues/21332.
+    @Override
+    @Test(enabled = false)
+    public void testLikePrefixAndSuffixWithChars() {}
+
+    /// array_cum_sum function in Presto C++ does not support decimal array inputs, see issue:
+    /// https://github.com/prestodb/presto/issues/24402.
+    @Override
+    @Test(enabled = false)
+    public void testArrayCumSumDecimals() {}
```

Comment:
I have seen a PR from Joe for this. It will be good to enable this test and assert the error messages so that he will have to explicitly make changes when his PR makes it to Velox.